Fix javadoc

bostko opened the pull request.

bostko committed:
Reduce poll intervals for tasks

bostko committed:
Fix javadoc

aledsage commented on src/main/java/org/jclouds/vcloud/director/v1_5/domain/CapacityWithUsage.java:
```diff
@@ -25,25 +25,24 @@
- *     &lt;/extension>
- *   &lt;/complexContent>
- * &lt;/complexType>
+ * &lt;complexType name="CapacityWithUsage"&gt;
```
@bostko Do we really need to use `&lt;` and `&gt;`? Can we not do the same as in https://github.com/apache/brooklyn-server/blob/master/core/src/main/java/org/apache/brooklyn/feed/ssh/SshFeed.java#L62-L63 (with the `<pre>` and `{@code ... }`? That seems to work for containing `<` and `>` in the javadoc.

aledsage committed:
Bump to jclouds 2.0.0 upstream dependency

aledsage committed:
Add “@Override”

(very useful when upgrading jclouds, so get errors if super-class
methods have disappeared).

aledsage committed:
Update TemplateBuilderImpl copy for jclouds 2.0.0

aledsage committed:
Fix errors for upgrade to jclouds 2.0.0

jclouds 2.0 uses google’s error-prone for additional validation checks.
This required quite a few changes, including:

- in pom.xml, change error-prone’s version to 2.0.15 and disable some 
  checks that couldn’t be found.
- in pom.xml, change “animal-sniffer” to check for java 1.7 compatibility, 
  rather than 1.6
- add generics (even where java can infer the right type)
- remove unreachable code
- disable VCloudDirectorComputeServiceAdapter.getAvailableVMsFromVAppTemplate,
  which was doing `computerName.equals(computerName)`. 
  TODO: What is the right fix?!
- Fields of enums must be final
- imports of classes must not use “static”
- remove @Provides from VCloudDirectorApi.getCurrentSession, because
  it’s not in a guice module.
- Use “L” instead of “l” for longs
- Add @Override everywhere
- Pass charset to `new String(byte[])`
- Make inner class `static` where possible
- Don’t throw exception in finally block
- Ensure `String.format(...)` has right number of args

googlielmo committed:
Revert to error-prone 2.0.5, which works w/ java 7

aledsage committed:
pom: tidy comments and update plugin versions

aledsage committed:
getAvailableVMsFomVappTemplate: remove unnecessary filter

aledsage committed:
Merge pull request #48 from cloudsoft/fix/jclouds-2.0-compatibility

Fix/jclouds 2.0 compatibility

bostko commented:
Needs to be revisited. Closing.
